saleor/graphql/product/mutations/products.py

```diff
@@ -311,8 +324,35 @@ def clean_input(cls, info, instance, input, errors):
             else:
                 cleaned_input['attributes'] = attributes
         clean_seo_fields(cleaned_input)
+        cls.clean_sku(product_type, cleaned_input, errors)
         return cleaned_input
 
+    @classmethod
+    def clean_sku(cls, product_type, cleaned_input, errors):
+        """Validate SKU input field.
+
+        When creating products that don't use variants, SKU is required in
+        the input in order to create the default variant underneath.
+        See the documentation for `has_variants` field for details:
+        http://docs.getsaleor.com/en/latest/architecture/products.html#product-types
+        """
+        if not product_type.has_variants and not cleaned_input.get('sku'):
+            cls.add_error(errors, 'sku', 'This field cannot be blank.')
```

Comment:
I think that we should validate if SKU is unique here as well, unless `clean_input` or `clean_instance` is already checking it.